contentcuration/contentcuration/frontend/accounts/pages/Create.vue

```diff
@@ -132,7 +132,7 @@
         </VSlideYTransition>
 
         <!-- Agreements -->
-        <Checkbox
+        <VCheckbox
```

Comment:
This should be changed to `Checkbox` in https://github.com/learningequality/studio/issues/4492